test/functional/wallet_bumpfee.py

```diff
@@ -128,6 +129,20 @@ def test_segwit_bumpfee_succeeds(rbf_node, dest_address):
     assert rbfid not in rbf_node.getrawmempool()
 
 
+def test_change_address_succeeds(rbf_node, peer_node):
+    dest_address = peer_node.getnewaddress()
+    change_address = rbf_node.getnewaddress()
+    hex = rbf_node.createrawtransaction([], {dest_address: '0.00100000'})
```

Comment:
The built-in function `hex` is redefined here. Choose another name :-)

Reply:
Done, now using `raw`.

Reply:
The built-in function `hex` is still redefined here, no? :-)